uvicorn/_types.py

```diff
@@ -0,0 +1,87 @@
+import sys
+from typing import Optional, Union
+
+if sys.version_info < (3, 8):
+    from typing_extensions import Literal, TypedDict
+else:
+    from typing import Literal, TypedDict
+
+# SCOPES
```

Comment:
I'm not sure we need these comments - it's quite readable what each item corresponds to, imo.

Reply:
yep, my rough way to mentally keep track on where I am, shouldnt be here